sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeTest.java

```diff
@@ -448,6 +448,21 @@ public void doNotShowUpdateSetupFailedDialogInBackground() {
         SharedPreferencesManager.remove(PREFERENCE_KEY_UPDATE_SETUP_FAILED_MESSAGE_KEY);
     }
 
+    @Test
+    public void doNotShowUpdateSetupFailedDialogInNullForegroundActivity() {
+        when(SharedPreferencesManager.getString(PREFERENCE_KEY_UPDATE_SETUP_FAILED_MESSAGE_KEY)).thenReturn("failed_message");
+
+        /* Trigger call. */
+        Distribute.getInstance().showUpdateSetupFailedDialog();
+        mActivity = null;
```

Comment:
I think we don't need this. But if we needed, I think it should be like this:
```
Distribute.getInstance().onActivityResumed(null);
```

and before we call `showUpdateSetupFailedDialog`.

Reply:
We will need to set the activity to null, I will do it through calling the on resume with null value

Reply:
Update: on resume call not needed, activity is already null if not set by on resume